src/components/Lyric/Lyric.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import ReactDOM from 'react-dom';
+import "./Lyric.scss";
+
+const Lyrics = (props) => {
+  return ReactDOM.createPortal(
+    (<div className="modal-overlay">
+    <section className="modal-content">
+     <p className="lyrics">{props.error || props.lyrics}<br/>some text</p>
+     <button onClick={props.closeLyrics}>Close</button>
```

Comment:
I think making this an X and putting it at the top right-hand side of the card would be slick 😎

Reply:
for sure, Should I just style that and add it to this PR? that was on my future iterations but thinking maybe I just do it today and put it on this PR

Reply:
Oh I didn't catch that on the PR 😅 No worries! We can get it on a future iteration!